platform/viewer/cypress/integration/volume/MPR.spec.js

```diff
@@ -14,8 +14,9 @@ describe('OHIF MPR', () => {
   });
 
   it('should go MPR for reconstructible displaySets and come back', () => {
-    cy.wait(1000);
+    cy.wait(250);
     cy.get(':nth-child(3) > [data-cy="study-browser-thumbnail"]').dblclick();
+    cy.wait(250);
```

Comment:
Currently intermittent without a bit of wait - would REALLY like to figure out how to read a value in the client to do this instead.